synth.py

```diff
@@ -7,12 +7,16 @@
 s.copy(templates, excludes=[
   '.eslintignore',
   '.eslintrc.yml',
+  '.github/release-please.yml',
+  '.github/workflows/',
+  '.github/publish.yml',
+  '.kokoro/*.*',
+  '.kokoro/**/*.*',
+  '.kokoro/**/**/*.*',
+  '.nycrc',
   '.prettierignore',
   '.prettierrc',
-  '.nycrc',
+  'codecov.yaml',
   'README.md',
-  '.github/release-please.yml',
-  '.kokoro/',
-  '.github/workflows/',
-  'codecov.yaml'
+  'renovate.json'
```

Comment:
We don't want our standard renovate config? Or are there special configs in the templated versions that don't apply here?

Reply:
We are supporting node.js *and* Go in this repo, so it has some customizations.  We're keeping the file, just not auto-updating.